### PR TITLE
Anonymizer - additional integration tests scenarios

### PR DESCRIPTION
**Pull request by navalev**
Added 2 integrationh tests to the anonimizer - most of the scenarions are covered in the unit tests under the invalid parameter error, resulting in 422. Here I added one edge case for a server error 500

### CODE REVIEW

File: functional-tests/tests/test_anonymizer.py [hunk 1]
```diff
@@ -50,6 +53,50 @@ def test_given_anonymize_called_with_empty_text_then_invalid_input_message_retur
...
+    """
+    response_status, response_content = anonymize(request_body)
+
+    expected_response = '{"error": "Invalid input, analyzer results can not be empty"}'
+    assert response_status == 422
+    assert equal_json_strings(expected_response, response_content)
+
+
+@pytest.mark.api
+def test_given_anonymize_called_with_deformed_body_than_internal_server_error_returned():
```
Comment (SharonHart) [inline functional-tests/tests/test_anonymizer.py:80]: then

**Review by balteravishay — DISMISSED** [review]

**Review by SharonHart — APPROVED** [review]

**Review by balteravishay — APPROVED** [review]
